### ISSUE TITLE
Qualified name availability in Resources and Data Sources

### ISSUE THREAD

**Issue by TerjeRusska** [Snowflake-Labs/terraform-provider-snowflake#2035]
**Is your feature request related to a problem? Please describe.**

Newer resources require the usage of `qualified name` which is not available in either existing resources or data sources. Example `snowflake_table_column_masking_policy_application` `snowflake_grant_privileges_to_role`.
Looping through these types of resources I have to build the qualified names myself from bits of available data.
```
resource snowflake_table_column_masking_policy_application masking_policy_sandbox {
  provider = snowflake.masking_admin
  for_each = local.masking_policy_sandbox

  # The qualified_name is in format "DATABASE"."SCHEMA"."TABLE"
  # FIXME: Take qualified_name from data source when it comes available
  table          = "\"${each.value.table.database}\".\"${each.value.table.schema}\".\"${each.value.table.name}\""
  column         = each.value.column
  masking_policy = "${each.value.table.database}.${each.value.table.schema}.MASKING_POLICY_${each.value.type}_${each.value.masking_policy}"
}
```

**Describe the solution you'd like**

Resources and Data sources should have qualified_name already available as a parameter

**Describe alternatives you've considered**

Currently manually building these qualified names

**Additional context**

**Comment by sfc-gh-jcieslak** [comment]
Hey @TerjeRusska 👋 
In the next release, all resources will have a `fully_qualified_name` computed field. We decided not to add it to the data sources, but stay tuned, and after the release, let us know what you think. The change should be out somewhere around next week.

**Comment by sfc-gh-jcieslak** [comment]
Hey @TerjeRusska 👋 
Yesterday we released version [v0.95.0](https://github.com/Snowflake-Labs/terraform-provider-snowflake/releases/tag/v0.95.0) of the provider that contains `fully_qualified_name` computed field for all resources (not applied to data sources).

**Comment by sfc-gh-jcieslak** [comment]
Closing due to long inactivity, from the emotes I assume you are happy with the results :)
